[EN/ID] Rewrite "Geki" and "Katu" (#4242)
ppy/osu-wiki: wiki/Katu/en.md

*See also: [Geki](/wiki/Geki)*

*Not to be confused with osu!taiko's Katu*

> **Katu (喝)**, *Katsu*, or *Beat!*, is a [scoring](/wiki/Score) term used when a player has completed a comboset without achieving the highest possible [accuracy](/wiki/Gameplay/Accuracy) on every note.

Reviewer: the same thing that I pointed out in "Geki": please use "combo" instead of "comboset"

> However, this can't be achieved by having a 50 or lower in the combo itself.

Reviewer: ```suggestion
However, Katu can't be achieved if a combo contains a 50 or a miss.
```

> There are two kind of Katu, which is based on the combo finishing judgment. A 300-ended Katu rewards more health bonus than 100-ended Katu, but always worse than a Geki.

Reviewer: tried to explain it a bit better
```suggestion
There are two types of Katu, which depend on the result of the last hit in a combo. The first type is worth 300 base score points, and the second one is worth 100 base score points. The 100-point Katu gives the least amount of HP, and 300-point Katu gives less HP than a [Geki](/wiki/Geki).
```

> This idea originates from the Nintendo DS game Elite Beat Agents which is one of the games the [osu!standard](/wiki/Game_mode/osu!) [game mode](/wiki/Game_mode) is based on.

Reviewer: ```suggestion
This idea originates from the Nintendo DS game [Elite Beat Agents](/wiki/iNiS_games), which [osu!](/wiki/Game_mode/osu!)'s gameplay is based on.
```

## Screenshots

> In here provided the screenshots of obtaining a Katu/Katsu/Beat from Elite Beat Agents, Ouendan 2, and osu! itself.

Reviewer: this sentence is unnecessary, please add captions to every image instead, something like this:
```markdown
![](img/eba-without-100k.jpg "Katu in Elite Beat Agents")
```

![](img/eba-without-100k.jpg) ![](img/oto-countdwn-300k.jpg)

![](img/osu-cutepnth-100k.jpg)

![](img/osu-stardisc-300k.jpg)

> 

Reviewer: these two screenshots need to be resized down to 1366×768

## Example gameplay links

- [Elite Beat Agents: Without a Fight/Jumpin' Jack Flash - YouTube](https://www.youtube.com/watch?v=wPOCmzY_fqs)
- [Osu! Tatakae! Ouendan 2: Countdown - YouTube](https://www.youtube.com/watch?v=6us1tY8jOSI)

## Other game modes

### osu!taiko

> Katu is the judgment given when you hit big notes at same time. osu!taiko have more leniency on handling both health bonus and score bonus for hitting both keys within short timespan.

Reviewer: I don't get the second sentence, mind explaining what it is about or rephrasing it?
```suggestion
Katu is the judgement given when a player hits big notes perfectly by pressing two keys of the same colour simultaneously. osu!taiko has more leniency on handling both health bonus and score bonus for hitting both keys within short timespan.
```

Author: i forgot to think the word "simulatenously" for no reason... thanks

### osu!catch

> Katu is counted for every droplets the player failed to catch. This value is not shown in result screen.

Reviewer: ```suggestion
Katu is counted for every droplet the player failed to catch. This value is not shown on the results screen.
```

### osu!mania

> On osu!mania, Katu is replaced with **200**. As the name states it gives 200 worth of score with slight penalty on the scoring side.

Reviewer: ```suggestion
On osu!mania, Katu is shown as a 200. As the value states, it gives 200 worth of score with slight penalty on the accuracy side.
```

## Other game aspects

### Storyboard (DS Games)

> Katu triggers the second-tier on the storyboard during the gameplay. Usually shown with natural spirit shown during the sequence.

Reviewer: not sure what you mean here as well
```suggestion
Katu triggers the second-tier on the storyboard during the gameplay, which usually shows a natural spirit during the sequence.
```

### Storyboard (osu!)

> Storyboard in osu! mostly intended for osu!standard. Obtaining Katu make several events occurs:

Reviewer: ```suggestion
Storyboards in osu! are mostly intended for osu!standard. Obtaining Katu makes several events occur:
```

> - Fail Layer activates.
> - Pass Layer deactivates.
> - Triggers "Failing" handler if transitioning from Pass state.

Reviewer: ```suggestion
- [Fail Layer](/wiki/Storyboard_Scripting/General_Rules#layers) is deactivated.
- [Pass Layer](/wiki/Storyboard_Scripting/General_Rules#layers) is activated.
- "Passing" event is triggered, if the previous state was Fail.
```

> To note that Fail side of storyboard in osu! is different than DS Games.

Reviewer: this is vague and gives no specific description, plus it is missing in the Geki article, mind removing it?

Author: this gives a clear difference from DS game actually. Geki in standard works the same as how it is on the DS.
Katu meanwhile is rather something in osu!'s implementation.

To explain more that on DS there are **3 layers** of succession rate.
- Bad (obtained if 50 or lower is in respective combo), shows how tasks are improperly done.
- Good (obtained if 100 or lower is in respective combo), on m980's authentic mapset, this side is shown on Fail side. Due to osu! SB trigger only checks 300 only combo or have some non-300 in it. (This assumes standard mode)
- Great (obtained if only 300 is in respective combo), not on this article for sure.

Reviewer: I see, but:

1) this is outside of the scope of these articles, and
2) this should go to the Storyboarding article or the likes

which is why you should exclude this specific sentence from this article

### osu!taiko

Katu is the judgement given when a player hits big notes perfectly by pressing two keys of the same colour simultaneously. osu!taiko has more leniency on handling both health bonus and score bonus for hitting both keys within short timespan.

### osu!catch

Katu is counted for every droplet the player failed to catch. This value is not shown on the results screen.

### osu!mania

> On osu!mania, Katu is shown as a 200. As the value states, it gives 200 worth of score with slight penalty on the accuracy side.

Reviewer: ```suggestion
In osu!mania, Katu is shown as a 200. As the value states, it gives 200 worth of score with slight penalty on the accuracy side.
```
